static/templates/django_registration/registration_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'layout.html' %}
+
+{% block content %}
+<table>
+    <form method='post' action=''>{% csrf_token %}
+        {{ form }}
+        <tr><td></td><td><input type="submit" value="Register" /></td>
```

Comment:
what is this tr td shit?

Reply:
lol, sorry, i just grabbed this from some reference with all the html called by `django-registration`, i'll get rid of the table